Firmware/Filament_sensor.cpp

```diff
@@ -74,40 +69,38 @@ void Filament_sensor::settings_init_common() {
         state = enabled ? State::initializing : State::disabled;
     }
 
-    autoLoadEnabled = eeprom_read_byte((uint8_t *)EEPROM_FSENS_AUTOLOAD_ENABLED);
-    runoutEnabled = eeprom_read_byte((uint8_t *)EEPROM_FSENS_RUNOUT_ENABLED);
     sensorActionOnError = (SensorActionOnError)eeprom_read_byte((uint8_t *)EEPROM_FSENSOR_ACTION_NA);
     if (sensorActionOnError == SensorActionOnError::_Undef) {
         sensorActionOnError = SensorActionOnError::_Continue;
     }
 }
 
-bool Filament_sensor::checkFilamentEvents() {
-    if (state != State::ready)
-        return false;
-    if (eventBlankingTimer.running() && !eventBlankingTimer.expired(100)) { // event blanking for 100ms
-        return false;
-    }
+void Filament_sensor::checkFilamentEvents() {
+    if ((state != State::ready) || (eventBlankingTimer.running() && !eventBlankingTimer.expired(100)))
```

Comment:
I think `(eventBlankingTimer.running() && !eventBlankingTimer.expired(100))`

can be simplified to `!eventBlankingTimer.expired_cont(100)`

Reply:
Done 👍